### PR TITLE
change log level, no more scary logs

### PR DESCRIPTION
**Pull request by c4pt0r**
change default log level.
`info` is too redundant.

### CODE REVIEW

**Comment by tennix** [comment]
docker-compose file specifies `-L=warn` for tidb-server which will override log level setting in config file. So please change it too in *docker-compose.yml* and *compose/templates/docker-compose.yml*

**Comment by c4pt0r** [comment]
@tennix PTAL

File: compose/templates/docker-compose.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ services:
       - --store=tikv
       - --path={{- template "pd_list" $ }}
       - --config=/tidb.toml
-      - -L=warn
+      - -L=error
```
Comment (tennix) [inline compose/templates/docker-compose.yml]: remove this and let tidb-server set log level from config file settings. And also change the default docker-compose.yml file in the root directory

Comment (c4pt0r) [reply compose/templates/docker-compose.yml]: oic

**Comment by c4pt0r** [comment]
@tennix  PTAL

**Comment by tennix** [comment]
use command line args or configuration file setting, not both

**Pull request merged**